src/test/optimizer/strategy/exists_reformulation_rule_test.cpp

```diff
@@ -140,10 +140,10 @@ TEST_F(ExistsReformulationRuleTest, QueryWithNotExists) {
 // Manually construct an exists query and apply the reformulation rule.
 // Compare with the manually created "optimal" plan and check for equality.
 TEST_F(ExistsReformulationRuleTest, ManualSemijoinLQPComparison) {
-  const auto parameter = opossum::expression_functional::parameter_(ParameterID{0}, node_table_a_col_a);
+  const auto parameter = opossum::expression_functional::correlated_parameter_(ParameterID{0}, node_table_a_col_a);
```

Comment:
I prefixed the parameter just as a workaround for the boost problem. We can remove that prefix now, am I right?

Reply:
Removed. But please don't add workarounds without discussing the underlying problem first. :/

Reply:
I think I was the one who said "Just put `opossum::expression_functional::` in there and forget about boost being stupid", so that one's on me.